apps/designer/app/routes/designer/$id.tsx

```diff
@@ -12,6 +12,10 @@ import * as db from "~/shared/db";
 import config from "~/config";
 import env from "~/env.server";
 import { ImagesUpload } from "~/designer/features/sidebar-left/types";
+import { s3UploadHandler } from "~/shared/uploads/s3-upload-handler";
+import { sentryException } from "~/shared/sentry";
+import { uploadToS3 } from "~/shared/uploads/upload-to-s3";
+import { uploadToDisk } from "~/shared/uploads/upload-to-disk";
```

Comment:
we should have all s3 specific logic one place and only use single interface everywhere else that is universal

Reply:
actually local upload is also one of the cases that should be behind a universal upload interface